packages/lending/src/protocols/aave-v3/lending-protocol.ts

```diff
@@ -78,6 +83,17 @@ export class LendingProtocol extends Protocol {
     this.hasNativeToken = hasNativeToken;
   }
 
+  async getReserveTokensFromCache() {
+    const reserveTokens: ReserveTokens[] = await fetchReservesData(this.id, this.chainId);
+    return reserveTokens;
```

Comment:
how about we `return await fetchReservesData(this.id, this.chainId)` directly?

Reply:
fixed, and we set to return type to `getReserveTokensFromCache`